sigma9.css

```diff
@@ -7,15 +7,17 @@
 
 @charset "utf-8";
 
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
I think we should try and self-host this, if we can. This import is bloated with versions of Inter that we might not even use. If we do choose to use the Multi-Axis Variable Font version of Inter, we wouldn't need to import any of the multiple standard font files or either of the single-axis font files.